packages/patternfly-4/react-core/src/components/Form/ActionGroup.js

```diff
@@ -23,13 +22,9 @@ const ActionGroup = ({ className, children, ...props }) => {
   const classesHorizontal = css(styles.formHorizontalGroup);
```

Comment:
This is unused with the removal of the isHorizontal switch.

Reply:
I _think_ you want to add that part back in. The actions area is broken in the horizontal form example. The `.pf-c-form__actions` element needs a `.pf-c-form__horizontal-group` parent to get the proper alignment in a horizontal form.

<img width="698" alt="screen shot 2019-02-13 at 5 09 16 pm" src="https://user-images.githubusercontent.com/35148959/52750547-2c929980-2fb2-11e9-94c1-2dfbab680830.png">